docker/Dockerfile.alpine

```diff
@@ -8,8 +8,8 @@ RUN addgroup -S zulipgr \
 USER zulip
 WORKDIR /home/zulip
 
-RUN set -ex; python3 -m venv zulip-terminal-venv \
-    && source zulip-terminal-venv/bin/activate \
+RUN set -ex; python3 -m venv zt_venv \
+    && source zt_venv/bin/activate \
```

Comment:
This doesn't achieve anything as you pointed out, but my goal was to have a similar venv naming scheme in both places - just for consistency. Feel free to drop this commit if you feel it might not be something significant - hence I added it in a separate commit as well.